src/options.rs

```diff
@@ -133,6 +133,14 @@ pub(crate) struct Config {
     #[structopt(short, long, help = "Base URL to check relative URls")]
     #[serde(default)]
     pub base_url: Option<String>,
+
+    #[structopt(
+        short = "B",
```

Comment:
Since it's such a specific feature, I'd rather skip the short argument. In my book it's better to spell that one out on the command-line to be explicit that we're passing credentials here.